frontend/app/(protected)/digital/activities/[id]/play/page.tsx

```
@@ -0,0 +1,494 @@
+import FillBlankCard from '@/components/activities/FillBlankCard';
+import TypingCard from '@/components/activities/TypingCard';
+import DrawingCard from '@/components/activities/DrawingCard';
+import AudioRecordingCard from '@/components/activities/AudioRecordingCard';
+import SessionSummary from '@/components/activities/SessionSummary';
+import { Button } from '@/components/ui/button';
+import { Progress } from '@/components/ui/progress';
+import { Card, CardContent } from '@/components/ui/card';
+import { Loader2, Pause, Play, SkipForward, ChevronLeft, HelpCircle } from 'lucide-react';
+import { useToast } from '@/hooks/use-toast';
+
+interface Card {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Card'. Consider to delete it or rename it.

'Card' is defined here:


(lint/suspicious/noRedeclare)